cmd/satellite/main.go

```diff
@@ -249,6 +263,79 @@ func cmdAPIRun(cmd *cobra.Command, args []string) (err error) {
 	return errs.Combine(runError, closeError)
 }
 
+func cmdRepairServiceRun(cmd *cobra.Command, args []string) (err error) {
+	ctx, _ := process.Ctx(cmd)
+	log := zap.L()
+
+	identity, err := runCfg.Identity.Load()
+	if err != nil {
+		zap.S().Fatal(err)
+	}
+
+	db, err := satellitedb.New(log.Named("db"), runCfg.Database)
+	if err != nil {
+		return errs.New("Error starting master database on satellite api: %+v", err)
+	}
+	defer func() {
+		err = errs.Combine(err, db.Close())
+	}()
+
+	pointerDB, err := metainfo.NewStore(log.Named("pointerdb"), runCfg.Config.Metainfo.DatabaseURL)
+	if err != nil {
+		return errs.New("Error creating metainfo database: %+v", err)
+	}
+	defer func() {
+		err = errs.Combine(err, db.Close())
+	}()
+
+	revocationDB, err := revocation.NewDBFromCfg(runCfg.Config.Server.Config)
+	if err != nil {
+		return errs.New("Error creating revocation database on satellite api: %+v", err)
+	}
+	defer func() {
+		err = errs.Combine(err, revocationDB.Close())
+	}()
+
+	tlsOptions, err := tlsopts.NewOptions(identity, runCfg.Server.Config, revocationDB)
+	if err != nil {
+		return err
+	}
+
+	dialer := rpc.NewDefaultDialer(tlsOptions)
```

Comment:
Create a proper peer for them such that the current configuration can be tested properly. @JessicaGreben should be able to help you out with this.

Reply:
Do you mean create something akin to the `satellite/peer.go`? I was thinking about making one in the repair package, but was wondering about circular dependencies by importing the `satellite.Config`, but I suppose if I created a new config struct that would work

Reply:
You would need to create something similar to `satellite/api.go`, hook it into testplanet etc.